examples/advanced_operations/add_smart_display_ad.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This adds a smart display campaign, an ad group, and a responsive display ad.
+
+More information about Smart Display campaigns can be found at:
+https://support.google.com/google-ads/answer/7020281
+"""
```

Comment:
Missing comment from golden https://github.com/googleads/google-ads-php/blob/3dc0d995dd292a3cb77671917da6528011f7a3d2/examples/AdvancedOperations/AddSmartDisplayAd.php#L66

Reply:
I've been using C# as a reference, as I prefer reading that over PHP, but I'll include this since the golden is the source of truth.